apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -105,7 +105,7 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
         if (this.advanceReceiptAdjustmentUpdatedData) {
             this.formatAdjustmentData(this.advanceReceiptAdjustmentUpdatedData.adjustments);
             this.advanceReceiptAdjustmentPreUpdatedData = cloneDeep(this.advanceReceiptAdjustmentUpdatedData);
-            this.adjustVoucherForm = this.advanceReceiptAdjustmentUpdatedData.adjustments.length ? cloneDeep(this.advanceReceiptAdjustmentUpdatedData) : this.adjustVoucherForm;
+            this.adjustVoucherForm = (this.advanceReceiptAdjustmentUpdatedData.adjustments && this.advanceReceiptAdjustmentUpdatedData.adjustments.length) ? cloneDeep(this.advanceReceiptAdjustmentUpdatedData) : this.adjustVoucherForm;
```

Comment:
Please remove this check as if the adjustments are empty then length would be 0 and in spite of initializing with empty array `[]` it will initialize with `adjustVoucherForm`